oauth/index.js

```diff
@@ -20,8 +20,9 @@ var _ = require('lodash');
 
 const authHeaderRegex = /Bearer (.+)/;
 const PRIVATE_JWT_VALUES = ['application_name', 'client_id', 'api_product_list', 'iat', 'exp'];
-const SUPPORTED_DOUBLE_ASTERIK_PATTERN = "**";
-const SUPPORTED_SINGLE_ASTERIK_PATTERN = "*";
+const SUPPORTED_TOKEN_ASTERISK_PATTERN = "/*/2/**";
```

Comment:
This is not supposed to '2' literally
It needs to be replaced by the text passed in the pattern

Reply:
Mentioned that in my PR comment. Just to pass the test atm so I'll remove for now.